addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18371,7 +18379,10 @@ msgctxt "#36209"
 msgid "Delete the databases for channels and guide and reimport the data from the backend afterwards."
 msgstr ""
 
-#empty string with id 36210
+#: system/settings/settings.xml
+msgctxt "#36210"
+msgid "Open the client priorities dialog, which allows modifying the priority for the enabled PVR clients, for instance to order channels client by client, according to your personal preferences. Lower value means higher priority"
```

Comment:
Could be `Opens the client priorities dialog which allows modifying the priority for enabled PVR clients according to your personal preferences, for instance to order channels client by client. Lower value means higher priority.`

Reply:
👍

Reply:
"Open" is okay, btw. All of those help texts are using direct speech.